### PR TITLE
CoCC recruitment post

### PR DESCRIPTION
**Pull request by kcranston**
Here is a draft blog post for the Code of Conduct Committee recruitment, with information about the community discussions later in June. Needs review from CoC committee: @malvikasharan @fmichonneau @karinlag @masamiy before merge.

### CODE REVIEW

File: _posts/2021/06/2021-06-09-recruiting-for-coc-committee.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: "Recruiting new members to the Code of Conduct Committee"
+date: 2021-06-09
+time: "00:00:00"
+tags: ["Community", "Code of Conduct"]
+---
+
+The Carpentries are committed to providing a welcoming and supportive environment for all people, and our [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html) (CoC) is the framework for this commitment. The [Code of Conduct Committee](https://carpentries.org/coc-ctte/) (CoCC) is responsible for managing the Code of Conduct, handling reports of conduct violations, and advocating for the CoC in the community.
+
+The Carpentries Code of Conduct Committee is currently recruiting new members. We seek 3-4 community members who are passionate about ensuring that The Carpentries is a welcoming space, who have time to dedicate to the committee, who bring diversity to the committee, and who (ideally) have experience developing, communicating, and enforcing conduct-related policy.
+
```
Comment (karinlag) [inline _posts/2021/06/2021-06-09-recruiting-for-coc-committee.md:14]: Suggest:

...commitee, and who bring diversity to the committee. In addition, it is beneficial, but not necessary to have previous experience with this kind of work.

**Comment by karinlag** [comment]
Added a small change, is good to go with or without it, IMO.

File: _posts/2021/06/2021-06-09-recruiting-for-coc-committee.md (hunk 1 above)
Comment (kcranston) [reply _posts/2021/06/2021-06-09-recruiting-for-coc-committee.md:14]: Yes! I like that you immediately identified the wording I was least happy with.

File: _posts/2021/06/2021-06-09-recruiting-for-coc-committee.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+The Carpentries are committed to providing a welcoming and supportive environment for all people, and our [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html) (CoC) is the framework for this commitment. The [Code of Conduct Committee](https://carpentries.org/coc-ctte/) (CoCC) is responsible for managing the Code of Conduct, handling reports of conduct violations, and advocating for the CoC in the community.
+
+The Carpentries Code of Conduct Committee is currently recruiting new members. We seek 3-4 community members who are passionate about ensuring that The Carpentries is a welcoming space, who have time to dedicate to the committee, and who bring diversity to the committee. In addition, it is beneficial, but not necessary, to have previous experience with this kind of work. 
+
+The Code of Conduct Committee will be leading two community calls in June to discuss the CoC, the committee, and answer questions about the open positions. These calls are in two different time zones to accommodate interested community members around the world.
+
+* Tuesday, June 22, 11AM UTC
+* Tuesday, June 22 8PM UTC
```
Comment (masamiy) [inline _posts/2021/06/2021-06-09-recruiting-for-coc-committee.md]:
```suggestion
* Tuesday, June 22, 8PM UTC
```

**Review by fmichonneau — APPROVED** [review]

**Comment by kcranston** [comment]
As requested by @fmichonneau , I've also added the following text to the top of the form: "We are collecting and processing personal data collected with this form in accordance with our Privacy Policy; https://docs.carpentries.org/topic_folders/policies/privacy.html".

**Comment by kcranston** [comment]
This has been reviewed by the CoC committee and ready to merge. @serahrono - in your capable hands now!

**Comment by serahkiburu** [comment]
splendid @kcranston! we will merge tomorrow ( June 9) cc @kmomar

**Pull request merged**